wcivf/apps/elections/templates/elections/includes/_previous_party_affiliations.html

```diff
@@ -0,0 +1,11 @@
+{% load static %}
+{% load i18n %}
+
+<i>
```

Comment:
I don't think the `<i>` tag is right here. It's designed for [_Idiomatic_ text](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i) and I don't fully understand what that means, but I don't think the whole section meets the criteria. I think it's ok to just have this in the normal text style?

Reply:
TIL about "[Idiomatic text](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i)"! Will fix that.